sktime/forecasting/base/_base.py

```diff
@@ -888,13 +901,20 @@ def _set_cutoff_from_y(self, y):
         -----
         Set self._cutoff to last index seen in `y`.
         """
-        y_mtype = mtype(y, as_scitype="Series")
+        # y_mtype = mtype(y, as_scitype="Series")
+
+        # danbartl: not sure why mtype inference does not work
```

Comment:
because the mtype you changed/extended is a `Panel` mtype, but this one tries to infer under the assumption that it is a `Series` mtype (these are different from the `Panel` ones). It should work if you do `as_scitype=`["Series", "Panel"]`, if you did everything right. Although I'm not sure whether that should be the target state.